api/src/client/DynamicsBusinessAddressClient.test.ts

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable unicorn/no-null */
+import { generateLicenseSearchAddress } from "@shared/test";
+import axios from "axios";
+import { BusinessAddressClient } from "../domain/types";
+import { LogWriter, LogWriterType } from "../libs/logWriter";
+import { DynamicsBusinessAddressClient } from "./DynamicsBusinessAddressClient";
+
+jest.mock("axios");
+jest.mock("winston");
+const mockAxios = axios as jest.Mocked<typeof axios>;
+
+describe("DynamicsBusinessAddressClient", () => {
+  let client: BusinessAddressClient;
+  let logger: LogWriterType;
+
+  let ORIGINAL_ORG_URL: string | undefined;
+  const ORG_URL = "www.test-org-url.com";
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    logger = LogWriter("NavigatorWebService", "ApiLogs", "us-test-1");
+    ORIGINAL_ORG_URL = process.env.DCA_DYNAMICS_ORG_URL;
+    process.env.DCA_DYNAMICS_ORG_URL = ORG_URL;
+
+    client = DynamicsBusinessAddressClient(logger, process.env.DCA_DYNAMICS_ORG_URL);
+  });
+
+  afterEach(() => {
+    process.env.DCA_DYNAMICS_ORG_URL = ORIGINAL_ORG_URL;
+  });
+
+  const mockAccessToken = "access-granted";
+  const mockBusinessIds = ["id2", "id3"];
+
+  it("makes a get request to dynamics api with business address query and auth token in header", async () => {
+    const businessAddressMockResponse = {
+      value: [
+        {
+          createdo: "2023-05-31T10:31:51Z",
+          rgb_city: "Washington",
+          rgb_county: null,
+          rgb_name: "Apt GFG 3900A Watson Pl NW",
+          rgb_state: "DC",
+          rgb_street1: "3900A Watson Pl NW",
+          rgb_street2: "Apt GFG",
+          rgb_typecode: 100000000,
+          rgb_zip: "20016-6739",
+          statecode: 0,
+          rgb_addressid: "ff166755-9eff-ed11-a81b-001dd80703e0",
+        },
+        {
+          createdon: "2023-05-31T10:32:04Z",
+          rgb_city: "Chesapeake",
+          rgb_county: null,
+          rgb_name: "Trlr T 322 Ballahack Rd",
+          rgb_state: "VA",
+          rgb_street1: "Trlr T 322 Ballahack Rd",
+          rgb_street2: null,
+          rgb_typecode: 100000001,
+          rgb_zip: "23322-2479",
+          statecode: 0,
+          rgb_addressid: "4da7455c-9eff-ed11-a81b-001dd80703e0",
+        },
+      ],
+    };
+
+    mockAxios.get.mockResolvedValue({ data: { ...businessAddressMockResponse } });
```

Comment:
[sand] could be written simpler as `{ data: businessAddressMockResponse }` I think